docs/source/jit.rst

```diff
@@ -434,8 +434,25 @@ Example (a type mismatch)::
             r = torch.rand(1)
         else:
             r = 4
-        return r # Type mismatch: r is set to type Tensor in the true branch
```

Comment:
Why was the comment removed?

Reply:
The error is shown with the block below

Reply:
okay wasn't sure if that showed up in the docs